client/src/components/App/Content/LandingPage/adminLandingPage/PostponedCard/PostponedCardStyles.ts

```diff
@@ -1,9 +1,11 @@
 import { makeStyles } from '@material-ui/styles';
 
+export const cardWidth = '14vw';
 export const cardHeight = '16vh';
 
 const useStyles = makeStyles({
     postponedCard: {
+        width: cardWidth,
```

Comment:
I think you should drop it and set width in the grid containing it instead (md)

Reply:
👍